src/NuGetGallery/Controllers/PackagesController.cs

```diff
@@ -2789,6 +2821,11 @@ public virtual async Task<JsonResult> CancelUpload()
         [ValidateAntiForgeryToken]
         public virtual async Task<JsonResult> PreviewReadMe(ReadMeRequest formData)
         {
+            var currentUser = GetCurrentUser();
+            if (_featureFlagService.AreEmbeddedReadmesEnabled(currentUser)) {
```

Comment:
Nit: please put curly braces on a new line:

```suggestion
            if (_featureFlagService.AreEmbeddedReadmesEnabled(currentUser))
            {
```